xwrf/io_plugin.py

```diff
@@ -73,6 +74,16 @@ def make_units_quantify_ready(dataset):
             dataset[var].attrs.pop('units', None)
 
 
+def modify_attrs_to_cf(dataset):
+    here = pathlib.Path(__file__).parent
+    attr_map = yaml.safe_load((here / 'cf_attr_map.yaml').read_text())
```

Comment:
How about we expose this `attr_map` in a standalone `config` object in a config.py module? This way, the user always has access to it, and we could provide a convenient way of modifying `attr_map` via the dictionary approach or by allowing the user to modify the `yaml` file directly. I'm imagining something along these lines

```python
xwrf.config.get('attr_map.ZNW')
```

```python
xwrf.config.set({'attr_map.ZNW.standard_name': 'foo_bar'})
```

This would be similar to what `dask` does with its configuration system

If it turns out that we need more configuration options for things other than the `attr_map`, we could easily manage all configurations via this one `config` interface.

Reply:
Another advantage of *attr_map in a standalone config object* is that we could add validations to this object via [Pydantic](https://pydantic-docs.helpmanual.io/) models and this would result in a better user experience in my opinion. 

Again, this depends on how many configurations we expect to provide to users. If `attr_map` is the only configurable object we plan to have in `xwrf`, then my suggestion would be overkill.

Reply:
Ccing @ncar-xdev/xwrf in case folks have thoughts on this...

Reply:
https://github.com/pytroll/donfig appears to be a good candidate for implementing a robust config system

Reply:
I'll have a look in the coming days and will try to implement this :)